runtime/tests/interpreter/memory_metering_test.go

```diff
@@ -80,7 +80,7 @@ func TestInterpretArrayMetering(t *testing.T) {
 		assert.Equal(t, uint64(25), meter.getMemory(common.MemoryKindAtreeArrayDataSlab))
 		assert.Equal(t, uint64(0), meter.getMemory(common.MemoryKindAtreeArrayMetaDataSlab))
 		assert.Equal(t, uint64(9), meter.getMemory(common.MemoryKindAtreeArrayElementOverhead))
-		assert.Equal(t, uint64(4), meter.getMemory(common.MemoryKindVariable))
+		assert.Equal(t, uint64(5), meter.getMemory(common.MemoryKindVariable))
```

Comment:
Where is the additional variable coming from?

Reply:
That is the `self` variable of the bound functions (which wasn't there earlier, but added as part of the changes).
i.e: https://github.com/onflow/cadence/blame/dd97d7424f4fa64a16c0e030cde363bb99018d6a/runtime/interpreter/interpreter.go#L547

Reply:
Ah, I see, makes sense 👍